napalm_logs/config/junos.yml

```diff
@@ -0,0 +1,24 @@
+# You should not use special characture in the value keys
+prefix:
+  values:
+    date: (\w+ \d\d)
+    time: (\d\d:\d\d:\d\d)
+    host: ([^ ]+)
+    processName: (\w+)
+    processId: (\d+)
+    error: (\w+)
+    message: (.*)
```

Comment:
I think the `.*` is implicit. At this point we build the prefix only. Everything follows after, will be considered as belonging to the core of the message. Thoughts?